docs/HACKING.md

```diff
@@ -24,3 +24,11 @@ pip install --user -e .
 ```console
 python3.7 setup.py test
 ```
+# Check the Report for Unit Test Coverage
```

Comment:
nit: Please add a newline before and after this header

Reply:
Hi John, thanks for your review! You are right, the format should be constant. It seems I have made too many commits about this minor change, I will be more cautious next time.